travel/services.py

```
@@ -0,0 +1,276 @@
+                        'type': 'connection_rejected',
+                        'match_id': str(match.id),
+                        'action_url': f"/matches"
+                    }
+                )
+    
+    def get_user_fcm_tokens(self, users):
+        """Get FCM tokens for users (placeholder - implement based on your user model)"""
+        # This assumes you have a UserProfile model or FCM token field
+        # You'll need to implement this based on your user model structure
+        tokens = []
+        for user in users:
```

Ruff (0.11.9)

Loop control variable `user` not used within loop body

Rename unused `user` to `_user`

(B007)